iodata/formats/gromacs.py

```diff
@@ -0,0 +1,148 @@
+# IODATA is an input and output module for quantum chemistry.
+# Copyright (C) 2011-2019 The IODATA Development Team
+#
+# This file is part of IODATA.
+#
+# IODATA is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# IODATA is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>
+# --
+"""gro file format.
+
+Files with the gro file extension contain a molecular structure in Gromos87 format.
+gro files can be used as trajectory by simply concatenating files.
+
+"""
+
+
+from typing import Tuple, Iterator
+
+import numpy as np
+
+from ..docstrings import (document_load_one, document_load_many)
+from ..utils import nanometer, picosecond, LineIterator
+
+
+__all__ = []
+
+
+PATTERNS = ['*.gro']
+
+
+@document_load_one('GRO', ['atcoords', 'atffparams', 'cellvecs', 'extra', 'title'])
+def load_one(lit: LineIterator) -> dict:
+    """Do not edit this docstring. It will be overwritten."""
+    frame_found = False
```

Comment:
Related to `_helper_read_frame return None` instead of exception: line can be removed.

Reply:
Good suggestion! Working on it.